code/src/components/Summary.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+const Summary = ({ name, volume, raceGroup, weapon, quest }) => {
+  return (
+    <section className="summary">
+      <h3>Summary:</h3>
+      <div className="summary-text">
+        <p>
+          Name: <span className="summary-styling">{name}</span>
+        </p>
+        <p>
+          Race: <span className="summary-styling">{raceGroup}</span>
+        </p>
+        <p>
+          Weapon of choise: <span className="summary-styling">{weapon}</span>
+        </p>
+        <p>
```

Comment:
Beautiful concept. I literally feel like I've been sent to some beautiful world * . * a nice idea with YT links